File: fitnesse-connect-test/src/main/java/com/processpuzzle/fitnesse/connect/test/domain/Car.java
package com.processpuzzle.fitnesse.connect.test.domain;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Car{
   @Id @GeneratedValue private long id;
   private String color;
   private String make;
   private String model;
   private int year;

   // constructurs
   Car() {}

   public Car( String make, String model, int year, String color ) {
      super();
      this.make = make;
      this.model = model;
      this.year = year;
      this.color = color;
   }

   // public accessors and mutators
   @Override public String toString() {
      return make + " " + model + " " + year + " " + this.color;
   }
   
   // properties
   // @formatter:off
   public String getMake() { return make; }
   public String getModel() { return model; }
   public int getYear() { return year; }
   public String getColor() { return color; }
   // @formatter:on
}

File: fitnesse-connect-testbed-ui/src/app/car/car.ts
export class Car {
   id: number;
   colour: string;
   description: string;
   make: string;
   model: string;
   imageUrl: string;
   year: string;

   constructor( make: string, model: string, description: string, imageUrl: string, year: string ){
      this.make = make;
      this.model = model;
      this.description = description;
      this.imageUrl = imageUrl;
      this.year = year;
   }
}
